packages/mgt-components/src/components/mgt-file-list/mgt-file-upload/mgt-file-upload.scss

```diff
@@ -25,36 +25,40 @@ $file-upload-dialog-primarybutton-background-color: set-var(
 $file-upload-button-background-color: set-var(upload__button__background-color, $theme-default, $file-upload);
 $file-item-background-color--hover: set-var(list__item__background-color--hover, $theme-default, $file-upload);
 $file-item-background-color--active: set-var(list__item__background-color--active, $theme-default, $file-upload);
-$file-upload-icon-button-background-color: var(--color, inherit);
+$file-upload-icon-button-background-color: var(--color, fileUploadIconBackground);
 $file-list-box-shadow: var(
   --file-list-box-shadow,
   0px 3.2px 7.2px rgba(0, 0, 0, 0.132),
   0px 0.6px 1.8px rgba(0, 0, 0, 0.108)
 );
 :host .file-upload-area-button {
   width: 100%;
+  --neutral-fill-hover: $file-upload-button-background-color;
+  --neutral-fill-rest: $file-upload-button-background-color;
+  --neutral-fill-active: $file-upload-button-background-color;
+  --neutral-fill-focus: $file-upload-button-background-color;
   div {
     float: var(--file-upload-button-float, right);
     text-align: center;
     display: table;
-    margin-top: 39px;
-    margin-right: 16px;
-    margin-left: 16px;
+    margin-top: 30px;
+    margin-right: 36px;
   }
 
   fluent-button.file-upload-button {
-    color: var(--file-upload-button-color, inherit);
+    color: var(--file-upload-button-color, $fileButtonColour);
```

Comment:
Adding this comment here as it's a close as I can get.

The conflict dialog buttons need some dark theme love:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/7122716/202544382-645e53dc-b4ec-43a7-b5fe-e586474dc44a.png">